Snowflake.Data/Snowflake.Data.csproj

```diff
@@ -26,7 +26,7 @@
     <PackageReference Include="Azure.Storage.Common" Version="12.12.0" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageReference Include="log4net" Version="2.0.12" />
-    <PackageReference Include="Portable.BouncyCastle" Version="1.8.10" />
+    <PackageReference Include="BouncyCastle.Cryptography" Version="2.2.1" />

```

Comment:
It was verified within https://github.com/snowflakedb/security_reviews/pull/4342